### PR TITLE
Downstream changes from Explorer UX upgrades

### PR DESCRIPTION
**Pull request by sarathms**
OONI Explorer is going through a major UX overhaul. There are some elements of the design system which are being redesigned. These will eventually land in other applications using them.

### CODE REVIEW

**Comment by elioqoshi** [comment]
> Use camel-case on buttons

Just little nitpicking that it's not camel-case but simply Title Case

File: components/atoms/Container.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+const Container = styled(Box)(maxWidth)
+
+Container.defaultProps = {
+  px: 3,
+  mx: 'auto',
+  maxWidth: [768, 1024, 1280]
+}
+
+export default Container
```
Comment (hellais) [inline components/atoms/Container.js]: This is cool. I like!

File: package.json [hunk 2]
```diff
@@ -19,27 +19,28 @@
...
+    "enzyme": "^3.9.0",
+    "enzyme-adapter-react-16": "1.5.0",
+    "enzyme-to-json": "^3.3.5",
     "eslint": "^4.19.1",
     "eslint-plugin-react": "^7.7.0",
-    "jest": "^23.6.0",
-    "jest-styled-components": "^6.2.0",
-    "react-test-renderer": "^16.5.2"
+    "jest": "^24.3.1",
+    "jest-styled-components": "^6.3.1",
```
Comment (hellais) [inline package.json]: @sarathms let's not forget to file a ticket about jest-styled-components and it blocking on us upgrading to styled-components v4.

**Comment by hellais** [comment]
I left a couple of comments on this PR, but it looks good to me and I would say it can be merged.

@sarathms do you need permissions to push to npm or do you have those already?

**Comment by sarathms** [comment]
I have published this package on npm before. I will go ahead.

File: package.json (hunk 2 above)
Comment (sarathms) [reply package.json]: Filed in #49

File: components/atoms/Container.js (hunk 1 above)
Comment (sarathms) [reply components/atoms/Container.js]: I put very little thought into what the values should be. They _sound_ reasonable, we need to make sure they _look_ appropriate too.

**Pull request merged**